deployment/windows/jellyfin.nsi

```diff
@@ -1,9 +1,9 @@
-; Shows a lot of debug information while compiling
-; This can be removed once stable.
-!verbose 4
-SetCompressor lzma
+!verbose 3
+SetCompressor /SOLID bzip2
```

Comment:
what's the reason for choosing a less efficient compressor, are we time-pressed during builds?

Reply:
It's quite annoying to wait ten minutes for the windows build to finish. This will at least shave off two minutes, which brings it closer to the second longest pipeline, which is around five minutes.

Reply:
The windows build step is nearly three times as long as the next longest pipeline, so it's really annoying waiting for it to finish every time.